File: Esempio di utilizzo file di DataBase Access con VB.NET/[C#]-Esempio di utilizzo file di DataBase Access con VB.NET/C#/Rubrica/ModificaDatiRubrica.cs
// Importazione dll .netFramework
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

// Spazio dei nomi Rubrica
namespace Rubrica
{
    // Classe ModificaDatiRubrica
    class ModificaDatiRubrica
    {
        // Dichiaro quattro variabili di tipo string ed assegno il valore String.Empty
        string nomeModificato = string.Empty;
        string cognomeModificato = string.Empty;
        string telefonoModificato = string.Empty;

        string posizioneRubrica = string.Empty;
        // Questa funzione quando richiamata restituisce il valore della variabile nomeModificato
        // Valorizzata tramite l'altro metodo ovvero setNomeModificato , il metodo sarà poi richiamato
        // Al momento di riassegnare il nome eventualmente modificato al campo NOME del Database Rubrica della riga selezionata
        // Sul datagridview
        public string getNomeModificato()
        {
            // Restituisco il valore di nomeModificato al chiamante
            return this.nomeModificato;
        }

        // Questa Sub quando richiamata si aspetta un parametro tipo string e
        // Valorizzata tramite il parametro nomemodificato,il metodo sarà poi richiamato
        // nel caso l'utente voglia modificare il campo NOME all'interno del Database Rubrica della riga selezionata sul datagridview e
        // Successivamente la variabile nomeModificato può essere recuperata tramite la funzione getNomeModificato()
        public void setNomeModificato(string nomemodificato)
        {
            // Valorizzo la variabile nomeModificato 
            this.nomeModificato = nomemodificato;
        }

        // Questa funzione quando richiamata restituisce il valore della variabile cognomeModificato
        // Valorizzata tramite l'altro metodo ovvero setCognomeModificato , il metodo sarà poi richiamato
        // Al momento di riassegnare il cognome eventualmente modificato al campo COGNOME del Database Rubrica della riga selezionata
        // Sul datagridview
        public string getCognonmeModoficato()
        {
            // Restituisco il valore di cognomeModificato al chiamante
            return this.cognomeModificato;
        }

        // Questa Sub quando richiamata si aspetta un parametro tipo string e
        // Valorizzata tramite il parametro cognomemodificato,il metodo sarà poi richiamato
        // nel caso l'utente voglia modificare il campo COGNOME all'interno del Database Rubrica della riga selezionata sul datagridview e
        // Successivamente la variabile cognomeModificato può essere recuperata tramite la funzione getCogNomeModificato()
        public void setCognomeModificato(string cognomemodificato)
        {
            // Valorizzo la variabile nomeModificato 
            this.cognomeModificato = cognomemodificato;
        }

        // Questa funzione quando richiamata restituisce il valore della variabile telefonoModificato
        // Valorizzata tramite l'altro metodo ovvero setNumeroTelefono , il metodo sarà poi richiamato
        // Al momento di riassegnare il telefono eventualmente modificato al campo TELEFONO del Database Rubrica della riga selezionata
        // Sul datagridview
        public string getNumeroTelefonoModificato()
        {
            // Restituisco il valore di telefonoModificato al chiamante
            return this.telefonoModificato;
        }

        // Questa Sub quando richiamata si aspetta un parametro tipo string e
        // Valorizzata tramite il parametro telefonodificato,il metodo sarà poi richiamato
        // nel caso l'utente voglia modificare il campo TELEFONO all'interno del Database Rubrica della riga selezionata sul datagridview e
        // Successivamente la variabile telefonoModificato può essere recuperata tramite la funzione getNumeroTelefonoModificato()
        public void setNumeroTelefono(string telefonomodificato)
        {
            // Valorizzo la variabile telefonoModificato 
            this.telefonoModificato = telefonomodificato;
        }

        // Questa funzione quando richiamata restituisce il valore della variabile posizioneRubrica
        // Valorizzata tramite l'altro metodo ovvero setPosizioneRubrica , il metodo sarà poi richiamato
        // Solamente per identificare tramite la clausola WHERE della query sql di modifica , in modo da
        // Modificare i dati appartenenti a tale posizione rubrica
        public string getPosizioneRubrica()
        {
            // Restituisco il valore di posizioneRubrica al chiamante
            return this.posizioneRubrica;
        }

        // Questa Sub quando richiamata restituisce il valore della variabile posizionerubrica
        // Valorizzata tramite l'altro metodo ovvero getPosizioneRubrica , il metodo sarà poi richiamato
        // Solamente per memorizzare e restituire tramite la funzione getPosizioneRubrica il numero di posizione rubrica che sara poi 
        // Confrontato tramite la clausola WHERE della query sql di modifica , in modo da
        // Modificare i dati apparteneti a tale posizione rubrica
        public void setPosizioneRubrica(string posizionerubrica)
        {
            // Valorizzo la variabile posizionerubrica 
            this.posizioneRubrica = posizionerubrica;
        }
    }
}

File: Esempio di utilizzo file di DataBase Access con VB.NET/[VB]-Esempio di utilizzo file di DataBase Access con VB.NET/VB/Rubrica/ModificaDatiRubrica.vb
' Classe ModificaDatiRubrica
Public Class ModificaDatiRubrica

    ' Dichiaro quattro variabili di tipo string ed assegno il valore String.Empty
    Dim nomeModificato As String = String.Empty
    Dim cognomeModificato As String = String.Empty
    Dim telefonoModificato As String = String.Empty
    Dim posizioneRubrica As String = String.Empty

    ' Questa funzione quando richiamata restituisce il valore della variabile nomeModificato
    ' Valorizzata tramite l'altro metodo ovvero setNomeModificato , il metodo sarà poi richiamato
    ' Al momento di riassegnare il nome eventualmente modificato al campo NOME del Database Rubrica della riga selezionata
    ' Sul datagridview
    Public Function getNomeModificato() As String
        ' Restituisco il valore di nomeModificato al chiamante
        Return Me.nomeModificato
    End Function

    ' Questa Sub quando richiamata si aspetta un parametro tipo string e
    ' Valorizzata tramite il parametro nomemodificato,il metodo sarà poi richiamato
    ' nel caso l'utente voglia modificare il campo NOME all'interno del Database Rubrica della riga selezionata sul datagridview e
    ' Successivamente la variabile nomeModificato può essere recuperata tramite la funzione getNomeModificato()
    Public Sub setNomeModificato(ByVal nomemodificato As String)
        ' Valorizzo la variabile nomeModificato 
        Me.nomeModificato = nomemodificato
    End Sub

    ' Questa funzione quando richiamata restituisce il valore della variabile cognomeModificato
    ' Valorizzata tramite l'altro metodo ovvero setCognomeModificato , il metodo sarà poi richiamato
    ' Al momento di riassegnare il cognome eventualmente modificato al campo COGNOME del Database Rubrica della riga selezionata
    ' Sul datagridview
    Public Function getCognonmeModoficato() As String
        ' Restituisco il valore di cognomeModificato al chiamante
        Return Me.cognomeModificato
    End Function

    ' Questa Sub quando richiamata si aspetta un parametro tipo string e
    ' Valorizzata tramite il parametro cognomemodificato,il metodo sarà poi richiamato
    ' nel caso l'utente voglia modificare il campo COGNOME all'interno del Database Rubrica della riga selezionata sul datagridview e
    ' Successivamente la variabile cognomeModificato può essere recuperata tramite la funzione getCogNomeModificato()
    Public Sub setCognomeModificato(ByVal cognomemodificato As String)
        ' Valorizzo la variabile nomeModificato 
        Me.cognomeModificato = cognomemodificato
    End Sub

    ' Questa funzione quando richiamata restituisce il valore della variabile telefonoModificato
    ' Valorizzata tramite l'altro metodo ovvero setNumeroTelefono , il metodo sarà poi richiamato
    ' Al momento di riassegnare il telefono eventualmente modificato al campo TELEFONO del Database Rubrica della riga selezionata
    ' Sul datagridview
    Public Function getNumeroTelefonoModificato() As String
        ' Restituisco il valore di telefonoModificato al chiamante
        Return Me.telefonoModificato
    End Function

    ' Questa Sub quando richiamata si aspetta un parametro tipo string e
    ' Valorizzata tramite il parametro telefonodificato,il metodo sarà poi richiamato
    ' nel caso l'utente voglia modificare il campo TELEFONO all'interno del Database Rubrica della riga selezionata sul datagridview e
    ' Successivamente la variabile telefonoModificato può essere recuperata tramite la funzione getNumeroTelefonoModificato()
    Public Sub setNumeroTelefono(ByVal telefonomodificato As String)
        ' Valorizzo la variabile telefonoModificato 
        Me.telefonoModificato = telefonomodificato
    End Sub

    ' Questa funzione quando richiamata restituisce il valore della variabile posizioneRubrica
    ' Valorizzata tramite l'altro metodo ovvero setPosizioneRubrica , il metodo sarà poi richiamato
    ' Solamente per identificare tramite la clausola WHERE della query sql di modifica , in modo da
    ' Modificare i dati appartenenti a tale posizione rubrica
    Public Function getPosizioneRubrica() As String
        ' Restituisco il valore di posizioneRubrica al chiamante
        Return Me.posizioneRubrica
    End Function

    ' Questa Sub quando richiamata restituisce il valore della variabile posizionerubrica
    ' Valorizzata tramite l'altro metodo ovvero getPosizioneRubrica , il metodo sarà poi richiamato
    ' Solamente per memorizzare e restituire tramite la funzione getPosizioneRubrica il numero di posizione rubrica che sara poi 
    ' Confrontato tramite la clausola WHERE della query sql di modifica , in modo da
    ' Modificare i dati apparteneti a tale posizione rubrica
    Public Sub setPosizioneRubrica(ByVal posizionerubrica As String)
        ' Valorizzo la variabile posizionerubrica 
        Me.posizioneRubrica = posizionerubrica
    End Sub
End Class
